src/components/ui/form/switch/Switch.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import { ChangeEvent, useCallback } from 'react';
+import styled, { css } from 'react-emotion';
+import { compose, setDisplayName } from 'recompose';
+import { RequiredInputComponentProps } from '../../../RequiredComponentProps';
+import {
+  withComponentTheme,
+  WithInnerComponentThemeProps,
+} from '../../../util/enhancers';
+import { Icon } from '../../icon';
+import { SwitchTheme } from './SwitchTheme';
+
+export interface SwitchProps
+  extends RequiredInputComponentProps<HTMLInputElement> {
+  checked: boolean;
```

Comment:
`checked` should be named `value`, to be consistent.

Maybe rename `value` to `inputValue`?

Reply:
What does `inputValue` signify?

I'm guessing the name checked comes from the checkbox DOM API https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#Additional_attributes